client/asset/btc/btc.go

```diff
@@ -437,13 +437,15 @@ type RPCWalletConfig struct {
 
 // WalletConfig are wallet-level configuration settings.
 type WalletConfig struct {
-	UseSplitTx       bool    `ini:"txsplit"`
-	FallbackFeeRate  float64 `ini:"fallbackfee"`
-	FeeRateLimit     float64 `ini:"feeratelimit"`
-	RedeemConfTarget uint64  `ini:"redeemconftarget"`
-	ActivelyUsed     bool    `ini:"special:activelyUsed"` //injected by core
-	WalletName       string  `ini:"walletname"`           // RPC
-	Birthday         uint64  `ini:"walletbirthday"`       // SPV
+	UseSplitTx             bool    `ini:"txsplit"`
+	FallbackFeeRate        float64 `ini:"fallbackfee"`
+	FeeRateLimit           float64 `ini:"feeratelimit"`
+	FeeRateLimitPerByte    uint64
+	FallbackFeeRatePerByte uint64
+	RedeemConfTarget       uint64 `ini:"redeemconftarget"`
+	ActivelyUsed           bool   `ini:"special:activelyUsed"` // injected by core
+	WalletName             string `ini:"walletname"`           // RPC
+	Birthday               uint64 `ini:"walletbirthday"`       // SPV
```

Comment:
Rather than `WalletConfig` pulling double duty, I suggest having a second struct

```Go
type baseWalletConfig struct {
	fallbackFeeRate  uint64 // atoms/byte
	feeRateLimit     uint64 // atoms/byte
	redeemConfTarget uint64
	useSplitTx       bool
}
```

and modifying `validateBtcWalletConf` to be something like...

```Go
func readBaseWalletConfig(walletCfg *WalletConfig) (*baseWalletConfig, error)
```

Set the defaults for zeros inside. Then you can embed `baseWalletConfig` in `baseWallet`.

```Go
type baseWallet {
    baseWalletConfig
    ...
}
```

and delete the existing fields.